### DISCUSSION TITLE
JK Bms  - allow multiple Bluetooth connections

### DISCUSSION THREAD

**Discussion by peregrines2** [Louisvdw/dbus-serialbattery#902] [General]
Hi!

Is there any way to allow multiple Bluetooth Connections to JK BMS? 
I would like to use dbus-Serialbattery (Bluetooth) and hass-victron at the same time.
But as soon as I add serialbattery I dont get any data for hass-victron anymore.

Thank You!

**Comment by mr-manuel** [comment]
No

**Reply by peregrines2** [reply]
Alright! Thank You Mr-Manuel for your answer & all your fantastic work!

**Reply by mr-manuel** [reply]
You could use hass-victron and then send the data to Venus OS via MQTT by using [dbus-mqtt-battery](https://github.com/mr-manuel/venus-os_dbus-mqtt-battery).

**Reply by peregrines2** [reply]
THANK You! Will try!
